spec/models/trainee_spec.rb

```diff
@@ -150,4 +150,31 @@
       it_behaves_like "a working search"
     end
   end
+
+  describe "#ordered_by_date" do
+    let(:trainee_one) { create(:trainee, updated_at: 1.hour.ago) }
+    let(:trainee_two) { create(:trainee, updated_at: 1.day.ago) }
+
+    it "orders the trainess by updated_at in descending order" do
+      expect(Trainee.ordered_by_date).to match_array([trainee_two, trainee_one])
+    end
+  end
+
+  context "trainee status" do
+    let(:trainee_one) { create(:trainee, state: "draft") }
+    let(:trainee_two) { create(:trainee, state: "submitted_for_trn") }
+    let(:trainee_three) { create(:trainee, state: "trn_received") }
```

Comment:
Just a style thing, but I'd name these variables something like `draft_trainee`, `trn_received_trainee` for readability. Then in your test cases, it's easier to check that we're testing for the correct trainees to be returned.

Not a blocker though!

Reply:
Updated